beaker/application.py

```diff
@@ -313,7 +313,7 @@ def dump(self, directory: str = "."):
         with open(os.path.join(directory, "contract.json"), "w") as f:
             if self.contract is None:
                 raise Exception("Contract empty")
-            f.write(json.dumps(self.contract.dictify()))
+            f.write(json.dumps(self.contract.dictify(), indent=4))
 
         with open(os.path.join(directory, f"{self.__class__.__name__}.json"), "w") as f:
```

Comment:
```suggestion
        with open(os.path.join(directory, f"application.json"), "w") as f:
```

Reply:
@barnjamin I brought in the change via https://github.com/algorand-devrel/beaker/pull/97/commits/f7fa21628ee245b6161efa4e8c79d6309b58e71f to account for generated changes.  Can you resolve if it looks good to you?